### PR TITLE
Implement #382 - Rework UI

### PR DESCRIPTION
**Pull request by lionel-nj**
**Summary:**

This PR provides support to rework UI.

**Expected behavior:** 

- [x] Add a dropzone for configuration file
  - [x] add support to transfer data to server side
    - [ ] test interface
    - [x] test server side
- [x] Add a button to launch the validator
  - [x] add support to launch process
    - [ ] test interface
    - [x] test server side
- [x] Add a visualisation of the configuration file
  - [x] add support to display json data
    - [ ] test interface
    - [x] test server side
- [ ] Add a visualisation of the validation report
  - [ ] add support to display beautifed json data 
    - [ ] test interface
    - [ ] test server side

Capture d’écran, le 2020-09-25 à 11 47 23

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `gradle test` to make sure you didn't break anything
- [x] Format the title like "Fix #<issue_number> - <short description of fix and changes>" (for example - "Fix #1111 - Check for null value before using field")
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

File: application/spring-boot-app/src/main/java/org/mobilitydata/gtfsvalidator/ServiceManager.java [hunk 1]
```diff
@@ -16,18 +16,279 @@
...
+     * @return the validation status as a String
+     */
+    public String validateFeed() throws IOException {
+        if (!isConfigInit()) {
+            throw (new IOException("Configuration file not provided"));
+        }
+        try {
+            // use case will inspect parameters and decide if help menu should be displayed or not
+            if (!config.printHelp().execute()) {
+
```
[the remark this answers is not in the document]
Comment (lionel-nj) [reply application/spring-boot-app/src/main/java/org/mobilitydata/gtfsvalidator/ServiceManager.java:83]: Yes, this is the springboot equivalent of our main cli-app

[the remark this answers is not in the document]
Comment (lionel-nj) [reply application/spring-boot-app/src/main/java/org/mobilitydata/gtfsvalidator/ServiceManager.java:83]: A few differences however: we return a string to identify the status of the validation: "Validation successfully executed" or the message from a potential exception that has been raised.

File: client/src/helper/Constants.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+export const Port = () => "8090";
+export const RunValidatorCommand = () => "/actions/runvalidator";
+export const GetReportContentCommand = () => "/actions/runvalidator";
```
[the remark this answers is not in the document]
Comment (lionel-nj) [reply client/src/helper/Constants.js]: There was a typo here, it has been fixed in the last commit

**Review by fabrice-v — APPROVED** [review]
Nice progress in a relatively short time, well done!

LGTM

**Pull request merged**